src/routes/Book.svelte

```diff
@@ -1,4 +1,6 @@
 <script>
+	import { Link, Route, Router } from 'svelte-routing';
```

Comment:
should not need Route and Router here, can always add later if we want to utilize within component